dataland-backend/src/main/kotlin/org/dataland/datalandbackend/model/lksg/LksgData.kt

```diff
@@ -0,0 +1,221 @@
+package org.dataland.datalandbackend.model.lksg
+
+import org.dataland.datalandbackend.annotations.DataType
+import org.dataland.datalandbackend.model.enums.commons.YesNo
+import java.math.BigDecimal
+import java.time.LocalDate
+
+/**
+ * --- API model ---
+ * Fields of the questionnaire for the LKSG framework
+ */
+@DataType("lksg")
+data class LksgData(
+    val dataDate: LocalDate? = null,
+
+    val lksgInScope: YesNo? = null,
+
+    val companyLegalForm: String? = null,
+
+    val vatIdentificationNumber: String? = null,
+
+    val numberOfEmployees: BigDecimal? = null,
+
+    val shareOfTemporaryWorkers: BigDecimal? = null,
+
+    val totalRevenue: BigDecimal? = null,
+
+    val totalRevenueCurrency: String? = null,
+
+    val responsibilitiesForFairWorkingConditions: YesNo? = null,
```

Comment:
In the excel specification I am counting 105 yes/no fields, in the data model here there are only 96. What is the reason for the discrepancy?

Reply:
Yes, you are right. Although I think when I was working on it these fields that are missing were for uploading the report and I was supposed to omit them. Now in Excel file, there are yes or no questions. I am adding them.

Reply:
62-70 on Jira in the file for upload and on newest Data Dictionary on Teams 62-70 now as a Yes or No question.